frontend/src/components/episode/episode.tsx

```diff
@@ -166,7 +167,7 @@ const Episode: React.FC = () => {
                   <div className={styles.shownotesWrapper}>
                     <h3>Time navigation</h3>
                     <ShownotesList
-                      shownotes={episode.shownotes}
+                      shownotes={getSortedShownotes(episode?.shownotes)}
```

Comment:
Would it be better to sort directly in the 'shownote-list' component?

Reply:
May be, but I'm not sure

Reply:
No, shownote-lists shouldn't know anything about sort